inference-engine/src/vpu/graph_transformer/src/stages/gather.cpp

```diff
@@ -13,18 +13,22 @@
 
 namespace vpu {
 
-void FrontEnd::parseGather(const Model& model, const ie::CNNLayerPtr& _layer, const DataVector& inputs, const DataVector& outputs) const {
-    IE_ASSERT(inputs.size() == 2);
-    IE_ASSERT(outputs.size() == 1);
-    auto layer = std::dynamic_pointer_cast<ie::GatherLayer>(_layer);
+void FrontEnd::parseGather(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const {
     IE_ASSERT(layer != nullptr);
+    IE_ASSERT(inputs.size() == 3);
```

Comment:
If you change this file, it would be better to replace `IE_ASSERT` with `VPU_THROW_UNLESS`, I think

Reply:
Fixed